lib/mastodon/cli/media.rb

```diff
@@ -151,7 +151,58 @@ def remove_orphans
           end
         end
       when :fog
-        fail_with_message 'The fog storage driver is not supported for this operation at this time'
+        paperclip_instance = MediaAttachment.new.file
```

Comment:
As pointed out by the linter, this is not actually used.

Reply:
Did not understand why that line also existed on S3 but thought there might be a reason for it. Removed it now.